worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitWorld.java

```diff
@@ -462,25 +462,9 @@ public com.sk89q.worldedit.world.block.BlockState getBlock(BlockVector3 position
     @Override
     public <B extends BlockStateHolder<B>> boolean setBlock(BlockVector3 position, B block, SideEffectSet sideEffects) {
         clearContainerBlockContents(position);
-        if (worldNativeAccess != null) {
-            try {
-                return worldNativeAccess.setBlock(position, block, sideEffects);
-            } catch (Exception e) {
-                if (block instanceof BaseBlock baseBlock && baseBlock.getNbt() != null) {
-                    LOGGER.warn("Tried to set a corrupt tile entity at " + position.toString()
-                        + ": " + baseBlock.getNbt(), e);
-                } else {
-                    LOGGER.warn("Failed to set block via adapter, falling back to generic", e);
-                }
-            }
-        }
-        if (WorldEditPlugin.getInstance().getLocalConfiguration().unsupportedVersionEditing) {
-            Block bukkitBlock = getWorld().getBlockAt(position.getBlockX(), position.getBlockY(), position.getBlockZ());
-            bukkitBlock.setBlockData(BukkitAdapter.adapt(block), sideEffects.doesApplyAny());
-            return true;
-        } else {
-            throw new RuntimeException(new UnsupportedVersionEditException());
-        }
+        Block bukkitBlock = getWorld().getBlockAt(position.getBlockX(), position.getBlockY(), position.getBlockZ());
+        bukkitBlock.setBlockData(BukkitAdapter.adapt(block), sideEffects.doesApplyAny());
+        return true;
```

Comment:
This completely breaks setting NBT and side-effect support in all Bukkit-based servers. The solution here _must_ not remove the calls to `worldNativeAccess`.

Reply:
Alright my team will be looking into this. Thanks for the feedback.

Reply:
I already have a proposed idea.